faros_event.sh

```diff
@@ -25,6 +25,10 @@ fi
 FAROS_GRAPH_DEFAULT="default"
 FAROS_URL_DEFAULT="https://prod.api.faros.ai"
 FAROS_ORIGIN_DEFAULT="Faros_Script_Event"
+FAROS_MAX_TIME_DEFAULT="10"
+FAROS_RETRY_DEFAULT="3"
+FAROS_RETRY_DELAY_DEFAULT="0"
```

Comment:
how about `FAROS_RETRY_DELAY_DEFAULT=1`?

Reply:
Sure, that's fine